cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/writer/NoOpMetadataServiceClient.java

```diff
@@ -42,4 +44,9 @@ public void update(MetadataMutation.Update updateMutation) {
   public void remove(MetadataMutation.Remove removeMutation) {
     // No Op
   }
+
+  @Override
+  public void batch(List<MetadataMutation> mutations) {
+
```

Comment:
empty body should have a "// no-op" comment. I am surprised checkstyle allows this.